frontend/src/lib/apiClient.js

```diff
@@ -65,6 +65,11 @@ const adminEditUser = (data, userId) => apiClient.patch(`api/v1/users/${userId}`
   email: data.email.value,
 });
 
+const userEditUser = (data, userId) => apiClient.patch(`api/v1/users/${userId}`, {
```

Comment:
new api call editCurrentUser instead of userEditUser in current user section. It takes first and last names and put it in user object. 
`const editCurrentUserName = (firstName, lastName) => apiClient.put('api/v1/auth',
  {
    user: {
      first_name: firstName,
      last_name: lastName,
    },
  },
  {
    headers: {
      Authorization: localStorage.jwt,
    },
  });`